Bash/KhanAcademy_Downloader/README.md

```diff
@@ -0,0 +1,54 @@
+# Khan Academy Downloader
```

Comment:
There are unncessary commits in this PR, 20 files have been changed, @HarshCasper I don't think merging this PR is a  good idea as of now.

@devRawnie It would be better if you make a new branch from master, and commit the files again.